app/src/main/res/layout-land/resume_lesson_fragment.xml

```diff
@@ -25,17 +25,15 @@
 
       <TextView
         android:id="@+id/resume_lesson_chapter_title_text_view"
+        style="@style/Heading1"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
```

Comment:
layout_width and layout_height can be removed from this and other files as it is `wrap_content`

Reply:
Done.